specification/communication/data-plane/Rooms/preview/2022-02-01-preview/communicationservicesrooms.json

```diff
@@ -495,69 +371,65 @@
     "RoomParticipant": {
       "description": "A participant of the room.",
       "required": [
-        "communicationIdentifier"
+        "rawId",
+        "role"
       ],
       "type": "object",
       "properties": {
-        "communicationIdentifier": {
-          "$ref": "#/definitions/CommunicationIdentifierModel"
+        "rawId": {
+          "description": "Raw id of the identifier.",
```

Comment:
```suggestion
          "description": "Raw ID representation of the communication identifier.",
```

Reply:
That aligns with the doc https://learn.microsoft.com/en-us/azure/communication-services/concepts/identifiers?pivots=programming-language-rest#raw-id-representation, so fine by me :-)
Maybe we could even link the article here?

Reply:
So will keep this description as we are planning not to use communication identifier.

Reply:
I wasn't necessarily referring to the `CommunicationIdentifierModel` from the Common swagger. I just wanted to be more explicit about what kind of identifier it is. (Even though you're not depending on the `CommunicationIdentifierModel` here, the rawId still a form of a "communication identifier" and you're still gonna use `CommunicationIdentifier` and derived classes to represent it in the SDK.)

Linking the docs above would help too. I like that idea.

Reply:
@petrsvihlik, thanks for the context. I was under impression its the same reason as not using CommunicationIdentiferModel in this spec. I have updated and added the doc link (sample below). Can you please take a 2nd look?

"raw id of the identifier. Please refer to the following document for additional information on Raw id.
https://learn.microsoft.com/en-us/azure/communication-services/concepts/identifiers?pivots=programming-language-rest#raw-id-representation"